vpr/src/place/timing_place_lookup.cpp

```diff
@@ -532,63 +532,48 @@ float delay_reduce(std::vector<float>& delays, e_reducer reducer) {
 
 /* Take the average of the valid neighboring values in the matrix.
  * use interpolation to get the right answer for empty blocks*/
```

Comment:
We should improve the comment. Here's a shot at comment to add:

We return the average placement estimated delay for a routing spanning (x,y).  We start with an averaging distance of 1 (i.e. from (x-1,y-1) to (x+1,y+1) and look for legal delay values to average; if some are found we return the average and if none are found we increase the distance to average over.

If no legal values are found to average over with a range of max_distance, we return IMPOSSIBLE_DELTA.